app/src/main/java/com/example/taskplanner/util/Progress.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.taskplanner.util
+
+enum class Progress {
+    Todo, InProgress, Done
```

Comment:
სავარაუდოდ ასე იმიტომ დაარქვი ამ ობიექტებს , რომ პირდაპირ თუსთრინგში გადაგეყვანა და ისე დაგესეტა ეკრანზე მარა უკეთესი მიდგომა იქნება Progress ენამ კლასს მისცე პარამეტრი ვთქვათ Value ან რამე ეგეთიდა თვითონ ობიექტები იყოს ასეთი ტიპის TODO("To Do") და როცა გამოყენება მოგინდება და ეკრანზე გამოტანა პირდაპირ მიწვდები project.progress.value და დაგიბრუნებს იმ სტრინგს რომელიც იქ გექნება

Reply:
ჩანაფიქრში მქონდა რო პროცესში გამომიჩნდებოდა რა იქნება უკეთესი და როგორ შემეძლება შეცვლა

Reply:
ხო ანუ იმიტომაც უცებ გამახსენდა რო მეც იგივე პრობლემა მქონდა მაგას რო ვაკეთებდი :D მარა ლოგიკურადაც რომ მიუდგე რამე value თუ გჭირდება ენამიდან პარამეტრი უნდა შეუქმნა რა იმიტორო გაძლევს მაგის საშუალებას ძალიან მარტივად

Reply:
done